ui_testing/pages/order_page.py

```diff
@@ -475,3 +513,38 @@ def get_order_id(self):
     def navigate_to_analysis_tab(self):
         self.base_selenium.click('order:analysis_tab')
         self.sleep_small()
+
+    def get_contact_field(self):
```

Comment:
ALso, contact feild not needed in LIMS-3406

Reply:
Done